### DISCUSSION TITLE
Add Internal IP for my server

### DISCUSSION THREAD

**Discussion by witzker** [Lissy93/dashy#1316] [Help and Support]
Hi,
I'm using Chrome extension and cannot find a way to add an Internal IP like 192.168.178.xx to the dashboard
pls explain how to achieve this
Many THX

**Comment by CrazyWolf13** [comment]
> Hi,
> I'm using Chrome extension and cannot find a way to add an Internal IP like 192.168.178.xx to the dashboard
> pls explain how to achieve this
> Many THX

If you mean this: https://chrome.google.com/webstore/detail/dashy-chrome-startpage-da/hadkmednchonkoigildhbbndeefhehmk

This is **not** associated with Lissy and the initial Dashy, it's community made so there is no guarantee it works.

I suggest just manually setting the start page in the browser settings to this url, works much better.

What is the point of this extension? Couldn't really read that off the description.

If that with the ip can't be solved for the moment I recommand nginxproxymanager selfhost, it would allow you to proxy for example 192.168.1.178:xx to something like dashy.myownserver.com, however this step requires a bit more knowledge and involves dns entrys, which may require setting up your server to act as a dns server for your whole network.
